extensions/webview/init.lua

```diff
@@ -17,14 +17,45 @@
 ---
 --- This module provides support for injecting custom JavaScript user content into your webviews and for JavaScript to post messages back to Hammerspoon.
 
-local module       = require("hs.webview.internal")
-module.usercontent = require("hs.webview.usercontent") ;
+local USERDATA_TAG = "hs.webview"
+
+local osVersion = require"hs.host".operatingSystemVersion()
```

Comment:
fwiw, I don't care, we have never been good at consistency, and I don't think this is the PR to start :)